webapp/channels/src/components/more_direct_bot_channels/index.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect} from 'react';
+import {useIntl} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {trackEvent} from 'actions/telemetry_actions.jsx';
+import {closeModal} from 'actions/views/modals';
+import GenericModal from 'components/generic_modal';
+import {isModalOpen} from 'selectors/views/modals';
+import {GlobalState} from 'types/store';
+import {ModalIdentifiers} from 'utils/constants';
+
+const MoreDirectBotChannelsModal = () => {
+    const dispatch = useDispatch();
+    const {formatMessage} = useIntl();
+
+    const show = useSelector((state: GlobalState) => isModalOpen(state, ModalIdentifiers.MORE_DIRECT_BOT_CHANNELS));
+
+    useEffect(() => {
+        trackEvent('plugins', 'more_direct_bot_channels_opened');
+    }, []);
+
+    const handleOnClose = () => {
```

Comment:
`more_direct_bot_channels` component is more placeholder at this point of the feature development. It will be fully added soon in https://mattermost.atlassian.net/browse/MM-51569